toolchain/lower/file_context.cpp

```diff
@@ -66,6 +70,22 @@ auto FileContext::Run() -> std::unique_ptr<llvm::Module> {
   return std::move(llvm_module_);
 }
 
+auto FileContext::BuildCompileUnit(llvm::StringRef module_name) -> void {
+  if (!include_debug_info_)
+    return;
+  // FIXME: Include directory path in the cu_file.
+  llvm::DIFile* cu_file = di_builder_.createFile(module_name, "");
+  // FIXME: Introduce a new language code for Carbon. C works well for now since
+  // it's something debuggers will already know/have support for at least.
+  // Probably have to bump to C++ at some point for virtual functions,
+  // templates, etc.
+  di_compile_unit_ = di_builder_.createCompileUnit(
+      llvm::dwarf::DW_LANG_C, cu_file, "carbon", false, "", 0, "");
```

Comment:
Is there value in labelling the last 4 parameters? I think it may make it easier for someone like me to understand the parameters without going to docs. That said, it looks like the last defaults to empty, so maybe can be removed.

```suggestion
      llvm::dwarf::DW_LANG_C, cu_file, "carbon", /*isOptimized=*/false, /*Flags=*/"", /*RV=*/0);
```